aries-core/src/js/components/helpers/Anchors/NavLink.js

```diff
@@ -3,20 +3,23 @@ import PropTypes from 'prop-types';
 import { Anchor, Text } from 'grommet';
 
 export const NavLink = ({ active, children, label, size, ...rest }) => {
-  const activeTextColor = active ? 'text' : null;
-  const defaultTextColor = 'text-xweak';
-  const fontWeight = 400;
-  const [textColor, setTextColor] = useState(defaultTextColor);
+  const textColor = 'text-strong';
+  const activeFontWeight = 700;
+  const defaultFontWeight = 400;
+  const [fontWeight, setFontWeight] = useState(defaultFontWeight);
```

Comment:
```suggestion
  const [fontWeight, setFontWeight] = useState(active ? activeFontWeight : defaultFontWeight);
```
If the link is active, it should start at the activeFontWeight.

Reply:
done.